src/components/UserPage.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from "react";
+import { connect } from "react-redux";
+import { Link, useParams, withRouter } from "react-router-dom";
+import { Container } from "reactstrap";
+import { getUserVideos } from "../store/actions";
+
+const UserPage = props => {
+  const { username } = useParams();
+
+  useEffect(() => {
+    props.getUserVideos(localStorage.getItem("token"), username);
+  }, [username]);
+
+  let greeting;
+  const getAllUserVideos = props.userVideos ? props.userVideos.length : 0;
```

Comment:
This looks like a new file, but this file also exists in the previous PR. That might be why it's showing up entirely as green here. It's also going to run into conflicts with master. 

With this ternary, it looks like it's saying "If props.userVideos exists, return props.userVideos.length, otherwise return 0". What is the thinking here?

Reply:
This is wrong, it is supposed to be 
```  const getAllUserVideos = props.userVideos ? props.userVideos: !props.userVideos; ```
I copied the ternary from a previous project I completed and I must've kept the .length and 0 in there.